docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -294,8 +294,8 @@ Here is a provisioning example using the JWT (Service Account key file) authenti
 apiVersion: 1
 
 datasources:
-  - name: Stackdriver
-    type: stackdriver
+  - name: Google Cloud Monitoring
+    type: cloud-monitoring
```

Comment:
```suggestion
    type: stackdriver
```

Reply:
good catch!

Reply:
I guess this is the only place where "stackdriver" is exposed to end users - that should be okay. But a but weird for those not having the stackdriver context. We maybe should add in the beginning of docs/readme like "Google Cloud Monitoring, formerly named Google Stackdriver, ..." or something. Thoughts?

Reply:
I hope it's fine to keep it at least here.

In any case, we should include renaming in the documentation. I have added it as a block quote.